src/libUtils/Logger.cpp

```diff
@@ -52,6 +54,10 @@ Logger::Logger(const char* prefix, bool log_to_file, streampos max_file_size) {
   this->m_logToFile = log_to_file;
   this->m_maxFileSize = max_file_size;
 
+  if (!boost::filesystem::exists(LOG_PATH)) {
```

Comment:
This check shouldn't be needed since `boost::filesystem::create_directory` already does a check to see if the path already exists. This is from the documentation of the method "Creation failure because p resolves to an existing directory shall not be treated as an error.". Thus if the path already exists `boost::filesystem::create_directory` will return false and no error will be reported.

Reply:
Nice catch, I'll remove the `exists` checking here. Thanks!